styleguide/preview.js

```diff
@@ -0,0 +1,8 @@
+import { Tooltip } from '../src/directives/index.js'
+
+// The export here MUST be default or module.export
+// this is what is imported by the styleguide
+export default (app) => {
+  app.directive('tooltip', Tooltip)
+}
+
```

Comment:
Should we also add OC/OCA here to the `globalProperties` lioke it was in the `Vue.prototype` before?

Reply:
Maybe, but I don't know. I think we try to wrap its usage in libraries. But we can of course still add it here once we encounter a usage.

Reply:
I meant this change:

https://github.com/nextcloud-libraries/nextcloud-vue/pull/4498/files#diff-a8c83951daf4c1d563181c22713c5162a6feae9effea0034e67bf9111c0b5b33

Reply:
Yes, I know. But OC and OCA is currently not used (directly) in the vue 3 version of nc/vue. The only place where we (will) use it is here:
https://github.com/nextcloud-libraries/nextcloud-vue/blob/3a9b9cd5f82aa571c0a94a9489f578f04d91faa8/src/components/NcAppSidebar/NcAppSidebarTabs.vue#L236

Maybe we cross that bridge (and add `OC` to vue) once we get there.